src/features/updateUser/index.ts

```diff
@@ -0,0 +1,2 @@
+export * as updateUserModel from './model/store';
```

Comment:
use public api

Reply:
Features slice - without index. Only folders with some features. That's info from FSD chat. It is not right?

Reply:
Not sure I understand you. Just add index inside model folder and export there what you need. Describe own api there

Reply:
fixed

Reply:
fixed